### PR TITLE
Updating what's new pages

### PR DESCRIPTION
**Pull request by hflynn**
See various cards in the 5.1 doc tasks column

### CODE REVIEW

**Comment by hflynn** [comment]
Further updates to come but if @jburel could check I have the ImageJ details correct, that would be handy, thanks!

File: omero/users/whatsnew.txt [hunk 1]
```diff
@@ -1,14 +1,21 @@
 What's new for OMERO 5.1
 ========================
 
-From a user's perspective, OMERO 5.1 introduces support for units and new more
-flexible user-added metadata. Other updates and new features include:
+Updates and new features for OMERO 5.1 include:
 
+- support for units
```
Comment (joshmoore) [inline omero/users/whatsnew.txt]: Could we do something catchier here by appending "....35 fields throughout the model like ... can now be stored in arbitrary units (e.g. micrometers)"

**Comment by hflynn** [comment]
--no-rebase

**Comment by jburel** [comment]
@hflynn: imageJ section looks good now.

**Comment by hflynn** [comment]
Going to leave this here and open a new PR for devs/sysadmin/endless breaking changes on Monday. Assuming @jburel & @joshmoore are happy now, this can be merged for m5 when Travis is happy.

**Pull request merged**